language/fr/firstlogin.lang

```diff
@@ -0,0 +1,22 @@
+<?php
+
+include("{$_EnginePath}language/en/firstlogin.lang");
```

Comment:
This file's translation is completed, therefore you can remove this default import.

Reply:
Done.